### PR TITLE
Update SDK to v1.1.0

### PR DESCRIPTION
**Pull request by foreza**

### CODE REVIEW

File: app/src/main/java/com/liveramp/ats/sampleapp/MainActivity.java [hunk 1]
```diff
@@ -62,14 +55,15 @@ protected void onCreate(Bundle savedInstanceState) {
     // Example on how to init LiveRamp
     private void initializeLiveRampATS(){
 
+
         if (notCheckingForCCPA || supportOtherGeos) {
-            LRAtsManager.INSTANCE.hasConsentForNoLegislation();
```
Comment (jasonc-lr) [inline app/src/main/java/com/liveramp/ats/sampleapp/MainActivity.java:66]: Note for the public: `hasConsentForNoLegislation` is a getter method, not a setter method. :)

File: app/src/main/java/com/liveramp/ats/sampleapp/MainActivity.java [hunk 2]
```diff
@@ -62,14 +55,15 @@ protected void onCreate(Bundle savedInstanceState) {
...
         }
 
-        // Notice: This will be removed in a subsequent SDK release
-        Configuration fallbackConfig = generateFallBackConfigurationForFileName("fallback_configuration.json");
-
-        LRAtsConfiguration config = new LRAtsConfiguration(appID, fallbackConfig);
+        // For the 2nd param (boolean), set "true" to enable test mode
+        // In test mode - the SDK will "simulate" envelopes.s
+        // Note: No network calls will actually be made!
+        LRAtsConfiguration config = new LRAtsConfiguration(appID, true);
```
Comment (jasonc-lr) [inline app/src/main/java/com/liveramp/ats/sampleapp/MainActivity.java:66]: Note for the public: setting this to `true` will cause the SDK not perform any network requests. Remember to set this to `false` before going to production!

**Review by jasonc-lr — APPROVED** [review]

**Comment by jasonc-lr** [comment]
Is this the equivalent of high-fiving yourself?

**Comment by foreza** [comment]
Yes.

**Pull request merged**